### PR TITLE
(chore) merge fork into here 

### PR DESCRIPTION
**Pull request by schristoff**

### CODE REVIEW

File: pkg/helm3/install_test.go [hunk 1]
```diff
@@ -72,6 +72,32 @@ func TestMixin_UnmarshalInstallAtomicExplicitTrue(t *testing.T) {
 	assert.True(t, *step.Atomic)
 }
 
+func TestMixin_UnmarshalInstallCreateNamespaceFalse(t *testing.T) {
+	b, err := ioutil.ReadFile("testdata/install-input-create-namespace-false.yaml")
```
Comment (troy0820) [inline pkg/helm3/install_test.go:76]:
```suggestion
	b, err := os.ReadFile("testdata/install-input-create-namespace-false.yaml")
```
nit: non-blocking

File: pkg/helm3/install_test.go [hunk 2]
```diff
@@ -72,6 +72,32 @@ func TestMixin_UnmarshalInstallAtomicExplicitTrue(t *testing.T) {
...
+	err = yaml.Unmarshal(b, &action)
+	require.NoError(t, err)
+	require.Len(t, action.Steps, 1)
+	step := action.Steps[0]
+
+	assert.False(t, *step.CreateNamespace)
+}
+
+func TestMixin_UnmarshalInstallCreateNamespaceExplicitTrue(t *testing.T) {
+	b, err := ioutil.ReadFile("testdata/install-input-create-namespace-true.yaml")
```
Comment (troy0820) [inline pkg/helm3/install_test.go:89]: same as above but non-blocking.

**Review by troy0820 — APPROVED** [review]
Nits: nothing blocking but 
LGTM!

File: pkg/helm3/install_test.go (hunk 2 above)
Comment (schristoff) [reply pkg/helm3/install_test.go:89]: we should put these comments to helm3 main since we are diverging TBD

**Pull request merged**